frontstage/common/message_helper.py

```diff
@@ -0,0 +1,81 @@
+import logging
+from datetime import datetime, date
+
+from structlog import wrap_logger
+
+logger = wrap_logger(logging.getLogger(__name__))
+
+
+def refine(message):
+    return {
+        'thread_id': message.get('thread_id'),
+        'subject': get_message_subject(message),
+        'body': message.get('body'),
+        'survey_id': message.get('survey'),
+        'ru_ref': get_ru_ref_from_message(message),
+        'from': get_from_name(message),
+        'sent_date': get_human_readable_date(message.get('sent_date')),
+        'unread': get_unread_status(message),
+        'message_id': message.get('msg_id')
+    }
+
+
+def get_message_subject(message):
+    try:
+        subject = message["subject"]
+        return subject
+    except KeyError:
+        logger.exception("Failed to retrieve Subject from thread")
+        raise
+
+
+def get_from_name(message):
+    try:
+        if message['from_internal']:
+            return "ONS Business Surveys Team"
+        msg_from = message['@msg_from']
+        return "{} {}".format(msg_from.get('firstName'), msg_from.get('lastName'))
```

Comment:
Should we be using the f"{variable}" format now? 🎨

Reply:
frontstage is on python 3.5, f-strings weren't introduced until python 3.6 :(  I've made a PR to start the process because all our services should be using the latest version where possible.